src/test/java/com/microsoft/sqlserver/jdbc/TestResource.java

```diff
@@ -201,5 +200,6 @@ protected Object[][] getContents() {
             {"R_objectNullOrEmpty", "The {0} is null or empty."},
             {"R_cekDecryptionFailed", "Failed to decrypt a column encryption key using key store provider: {0}."},
             {"R_connectTimedOut", "connect timed out"},
+            {"R_queryCancelled", "The query was cancelled."},
```

Comment:
wrong spelling

Reply:
Oops, that would cause the test to fail.